std_msgs/String ssid
std_msgs/String signal
std_msgs/String quality
std_msgs/String last_beacon
std_msgs/String mac_address
nav_msgs/Odometry odom